spec/services/export/spending_breakdown_spec.rb

```diff
@@ -284,5 +284,47 @@ def value_for_header(header_name)
         expect(value_for_header("Tags")).to eq("Ayrton Fund|Double-badged for ICF")
       end
     end
+
+    context "when there's no organisation" do
+      subject { described_class.new(source_fund: @source_fund) }
+
+      it "includes fund and programme activities" do
+        expect(subject.rows.count).to eq(5)
+
+        count_by_level = subject.rows.each_with_object(Hash.new(0)) do |row, hash|
+          level = row[3]
+          hash[level] += 1
+        end
+
+        # Because of the way things are set up in `before(:all)`, we have three
+        # funds but they all have the same source fund code, hence why we get
+        # three here even when filtering by source fund
+        expect(count_by_level["Fund (level A)"]).to eq(3)
+        expect(count_by_level["Programme (level B)"]).to eq(1)
+        expect(count_by_level["Project (level C)"]).to eq(1)
+      end
+
+      it "includes activities for all organisations" do
+        activity_from_different_organisation = create(:project_activity)
+
+        # Five are created in `before(:all)` - three by `@activity`, one by each
```

Comment:
But they were, all of them, deceived, for another was made, in the land of Mordor-